<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingPage</class>
 <widget class="QWidget" name="settingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>      背景音乐</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_back">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/setting/slider-track-1.png);
border-image: url(:/image/setting/slider-track-1.png);
QSlider::handle:horizontal {
     /* background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);*/
	border-image: url(:/image/setting/slider-track-2.png);
      /*border: 1px solid #5c5c5c;*/
      width: 28px;
      margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
      /*border-radius: 3px;*/
  }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Value1">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;华文行楷&quot;;
border-image: url(:/image/setting/score.png);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Enter">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>710</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;华文行楷&quot;;

border-image: url(:/image/setting/Ok.png);</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
